apps/builder/app/builder/features/style-panel/sections/transitions/transition-content.tsx

```diff
@@ -50,25 +50,18 @@ export const TransitionContent = ({
     IntermediateStyleValue | InvalidValue | undefined
   >({ type: "intermediate", value: transition });
 
+  useEffect(() => {
```

Comment:
wouldn't this effectively overwrite what user is typing if some value comes from outside?

Reply:
can you tell me what this code is for?

Reply:
I can't figure out when is this actually needed

Reply:
I feel stupid, I can't see why this code is needed, so I did the dumb thing - deleted it, so we don't merge stuff that isn't doing anything, if it did do something, I am sure you know what and we can bring it back